src/components/PreviewScreen.jsx

```diff
@@ -121,6 +123,7 @@ const PreviewScreen = React.memo(({ authTokenByRoomCodeEndpoint }) => {
               onJoin={onJoin}
               token={token}
               asRole={previewAsRole}
+              userId={userId}
```

Comment:
why is this change needed ?

Reply:
I was testing out if the user id was available on the preview screen so wanted to pass it to the UI. Will remove this.